@noun-auction/components/AuctionUi/SettleAuction/SettleAuction.tsx

```diff
@@ -0,0 +1,24 @@
+import { auctionWrapperVariants } from '@noun-auction/styles/NounishStyles.css'
+import { StackProps } from '@zoralabs/zord'
+
+import { isOGNounAddress } from '../NounsBidForm'
+import { SettleAuctionBuiler } from './SettleAuctionBuilder'
+import { SettleAuctionOGNouns } from './SettleAuctionOGNouns'
+
+export interface SettleAuctionProps extends StackProps {
+  useErrorMsg?: boolean
+  auctionContractAddress: string
+  layout: keyof typeof auctionWrapperVariants['layout']
+}
+
+export function SettleAuction({ auctionContractAddress, ...rest }: SettleAuctionProps) {
+  if (isOGNounAddress(auctionContractAddress)) {
+    return (
+      <SettleAuctionOGNouns auctionContractAddress={auctionContractAddress} {...rest} />
+    )
+  } else {
+    return (
+      <SettleAuctionBuiler auctionContractAddress={auctionContractAddress} {...rest} />
```

Comment:
Needs to be renamed to SettleAuctionBuilder

Reply:
+